tests/manager/manager-executeQuery-001.phpt

```diff
@@ -73,7 +73,7 @@ object(MongoDB\Driver\Cursor)#%d (%d) {
       ["mode"]=>
       int(1)
       ["tags"]=>
-      array(0) {
+      object(stdClass)#%d (0) {
```

Comment:
@bjori: When `mongoc_read_prefs_t` is initialized, its tags are also initialized to an empty BSON document. In turn, `php_phongo_read_preference_to_zval()` will do:

```
if (read_prefs->tags.len) {
    php_phongo_bson_state  state = PHONGO_BSON_STATE_INITIALIZER;

    MAKE_STD_ZVAL(state.zchild);
    bson_to_zval(bson_get_data(&read_prefs->tags), read_prefs->tags.len, &state);
    add_assoc_zval_ex(retval, ZEND_STRS("tags"), state.zchild);
} else {
    add_assoc_null_ex(retval, ZEND_STRS("tags"));
}
```

That `else` condition will really never be reached. Would you rather I change the condition to `read_prefs->tags.len > 5` and add a comment that we only dump it if it's a non-empty document? Otherwise, I can leave this as-is.